frontend/dockerfile/dockerfile_test.go

```diff
@@ -6562,7 +6562,7 @@ FROM scratch
 COPY --from=0 / /
 `)
 
-	const expectedDigest = "sha256:14b7856c37777ad7e8f4a801cb98abec98c5ae0f4a8dbb152447d18f1c1a3ba9"
+	const expectedDigest = "sha256:d286483eccf4d57c313a3f389cdc196e668d914d319c574b15aabdf1963c5eeb"
```

Comment:
Why changed?

Reply:
Apparently, that digest [may change based on snapshotter version and other variables](https://github.com/moby/buildkit/blob/6d7ef59e71aa4ce2e614b0a81de5a3bf73a57284/frontend/dockerfile/dockerfile_test.go#L6601).

Reply:
Which may have happened because we update containerd. Seems like last time this was changed was when containerd was updated to 1.7.0-rc3: https://github.com/moby/buildkit/commit/98deacf59c349975d48cfce1a9f09f6e010fc7eb#diff-93731187c2a292b7e92b852225675f88b41b7634dafd66cfeab50697bd806ddbR6561

Reply:
Still curious to know which commit introduced this change

Reply:
😭 

As long as we have this test we need to find the exact cause of these changes.

If this breaks with every containerd release we need to consider forking the differ and starting to maintain our own one that we keep stable(or define "differ compatibility version" that can be set by user).

Reply:
It seems that the difference is due to the fact that the resulting OCI tarball no longer has a ```manifest.json``` inside of it. 

With the current master branch, it seems we still have a ```manifest.json``` even if we [explicitly specify](https://github.com/moby/buildkit/blob/master/exporter/oci/export.go#L195-L196) we want an ```oci``` image, which should skip adding the docker manifest. This seems to have been fixed in https://github.com/containerd/containerd/pull/8213. A fix which this update pulls in.

So it seems this update also fixes a bug :smile:.

Apologies for the late reply on this. I have been traveling for the paste couple of weeks.

Reply:
Here are the contents of the ```master``` branch vs this branch (the blobs folder is omitted):

The master branch:

```bash
root@buildkit:~/test/b# sha256sum $HOME/test/a/ex/*
sha256sum: /root/test/a/ex/blobs: Is a directory
3fc4c7275c73ef20e8b5756d54e0e4dbd12b8df91505410c5e9ff55957c06007  /root/test/a/ex/index.json
4be49e4e12d735ac51e8acd43a62ca3c2135f66d86f1f6b419aae17c8d507b28  /root/test/a/ex/manifest.json
18f0797eab35a4597c1e9624aa4f15fd91f6254e5538c1e0d193b2a95dd4acc6  /root/test/a/ex/oci-layout
```

This branch:

```bash
root@buildkit:~/test/b# sha256sum $HOME/test/b/ex/*
sha256sum: /root/test/b/ex/blobs: Is a directory
3fc4c7275c73ef20e8b5756d54e0e4dbd12b8df91505410c5e9ff55957c06007  /root/test/b/ex/index.json
18f0797eab35a4597c1e9624aa4f15fd91f6254e5538c1e0d193b2a95dd4acc6  /root/test/b/ex/oci-layout
```